src/index.html

```diff
@@ -10,13 +10,110 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header-wrapper">
+          <a
+            href="#"
+            class="logo"
+          >
+            <img
+              src="./images/logo.png"
+              alt="Moyo logo"
+              class="logo__img"
+            />
+          </a>
+
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link is-active"
+                >
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
              </li>
              
              <li class="nav__item">
```